Disable the tests because they take several hours to run.

diff -ruN openldap-2.4.28/debian/rules openldap-2.4.28.new/debian/rules
--- openldap-2.4.28/debian/rules	2013-02-20 12:13:07.300717163 -0800
+++ openldap-2.4.28.new/debian/rules	2013-02-20 12:12:31.000000000 -0800
@@ -179,3 +179,5 @@
 		contrib/slapd-modules/autogroup/autogroup.lo \
 		contrib/slapd-modules/autogroup/autogroup.la \
 		contrib/slapd-modules/autogroup/autogroup.o
+
+override_dh_auto_test:
